src/arrays/arrays_with_methods/medium/q6/MainTest.java

```diff
@@ -0,0 +1,31 @@
+package arrays.arrays_with_methods.medium.q6;
+
+import global.tools.CustomAssertions;
+import global.utils.MethodUtil;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+public class MainTest {
+    // Parsons with distractors
+
+    static Stream<Arguments> inputProvider() {
+        return Stream.of(
+                Arguments.of(new int[]{5, 4, 10, 1, 7}, new int[]{10, 1}),
+                Arguments.of(new int[]{-3, -14, -8, -23, -28, -1, -4}, new int[]{-1, -28}),
```

Comment:
In the question we said to assume the array values are between 0 and 100, so we shouldn't test values that aren't